testing/acceptance_test.py

```diff
@@ -557,6 +558,67 @@ def join_pythonpath(what):
             "*not*found*test_hello*",
         ])
 
+    def test_cmdline_python_namespace_package(self, testdir, monkeypatch):
+        monkeypatch.delenv('PYTHONDONTWRITEBYTECODE', False)
```

Comment:
Nitpick: would you mind using a keyword argument here (`raising=False`)? I find keyword arguments for boolean parameters greatly improve readability. :grin: